### ISSUE TITLE
different modules installed vs displayed in colab

### ISSUE THREAD

**Issue by preyasgarg** [fastai/fastai#1701]
**Describe the bug**

I am working on colab and I am getting an error while implementing WGAN notebook in colab.

while calling learn.fit it is giving an error when it calls a new function in callback, upon reviewing the code i found out that code is different in google colab vs on GitHub.
 how can I fix this

**Provide your installation details**

**To Reproduce**
I just ran the wgan notebook and error appeared
**Expected behavior**

**Screenshots**

![screenshot 71](https://user-images.githubusercontent.com/37271898/53223404-42c9e500-3697-11e9-8758-3ebbfceb8d10.png)

I also tried to uninstall and then reinstalled it but problem persists

**Comment by kechan** [comment]
i think colab is on 1.0.45  (!pip list |grep -i fast). Whats on GitHub is latest greatest breeding edge. 
if you really want that, you have to git clone to your colab local session, then do

!pip install -e {local_fastai_repos_path}

Warning: picking things directly up from GitHub is also hazardous as it isn't a release and maybe unstable. try and see if  this unblocks u.

**Comment by preyasgarg** [comment]
Thanks I knew that and already done that but the intent was as to why pip install fastai or default fastai library at colab isn't working correctly.
It was a library bug so I posted it here.

**Comment by kechan** [comment]
I see. It is at least good to know it works with the GitHub version., so something is already fixed.

**Comment by sgugger** [comment]
There is no bug here: we were informed of that bug recently, fixed it in master, and it will be in the next released version (v1.0.46) of fastai. In the meantime, you have to user master as @kechan pointed out if you want the fix.
